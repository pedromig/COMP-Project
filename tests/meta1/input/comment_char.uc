/* gagagagag
'\t'
'           '
*/
//	//
	*/
/*
gagagaagga
'           '
*/
//
'\t'
//ulullulu
'o\n';
/t
/* //jfskdfofwe
//jfkwsemf*/
'\n'
/*

**/
