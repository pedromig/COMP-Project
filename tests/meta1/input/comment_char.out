MUL
DIV
CHRLIT('\t')
Line 14, col 1: invalid char constant ('o\n')
SEMI
DIV
ID(t)
CHRLIT('\n')
